src/components/skills-quiz/SelectedJobSkills.jsx

```diff
@@ -17,8 +17,7 @@ const SelectedJobSkills = () => {
   selectedJobSkills = selectedJobSkills?.slice(0, 5);
 
   return (
-    <div style={{ paddingLeft: '10%' }} className="mb-3 mt-5">
-      <h4>Top Skills for {selectedJob}</h4>
```

Comment:
Is it a requirement to remove this heading?

Reply:
> https://www.figma.com/file/OYth02FHxzgy06yy1DMljD/Skills-Builder-Handoff?node-id=0%3A1
> 
> here is the link to how the output should look like

We should still confirm with Nathan if it's a requirement to remove "Top skills for <selected job>"

Reply:
according to Kevin, "because the heading above indicates “jobs and skills” we should be safe to remove."